### DISCUSSION TITLE
bottom sheet modal

### DISCUSSION THREAD

**Discussion by pouyarezvani** [software-mansion/react-native-screens#963] [Q&A]
@WoLewicki  Is it possible to achieve a `bottom-sheet` / half `modal` feature within the `react-native-screens` library? 
Currently, every modal within `stackPresentation` does not let me achieve this. 
Any code examples anybody has done to achieve this?

more specifically I'm referring to the bottom sheet that is in the iOS map App.
![Image from iOS](https://user-images.githubusercontent.com/32917154/120583986-ffb55b00-c3e3-11eb-98df-5e786743b126.gif)

Any help would be appreciated!

**Comment by WoLewicki** [comment]
It is not possible now, but we are open to PRs anytime :tada:
